src/s3fs_util.cpp

```diff
@@ -1151,6 +1151,10 @@ void show_help (void)
     "   curldbg - put curl debug message\n"
     "        Put the debug message from libcurl when this option is specified.\n"
     "\n"
+    "   cipher_suites - customize TLS cipher suite list\n"
+    "        Customize the list of TLS cipher suites.\n"
+    "        Expects a colon separated list of cipher suite names.\n"
```

Comment:
We need some kind of documentation so that users know how to interact with this flag.  If curl or the cipher suite can emit all valid ciphers, let's do that.  Otherwise link to the relevant documentation like curl does:

```
       --ciphers <list of ciphers>
              (SSL) Specifies which ciphers to use in the connection. The list of ciphers must specify valid ciphers.
              Read up on SSL cipher list details on this URL: http://www.openssl.org/docs/apps/ciphers.html

              NSS ciphers are done differently than OpenSSL and GnuTLS. The full list of NSS ciphers is in the NSSCi‐
              pherSuite entry at this URL: http://git.fedora‐
              hosted.org/cgit/mod_nss.git/plain/docs/mod_nss.html#Directives

              If this option is used several times, the last one will be used.
```